internal/testutils/pickfirst/pickfirst.go

```diff
@@ -0,0 +1,72 @@
+/*
+ *
+ * Copyright 2022 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package pickfirst contains helper functions to check for pickfirst load
+// balancing of RPCs in tests.
+package pickfirst
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/peer"
+	"google.golang.org/grpc/resolver"
+
+	testgrpc "google.golang.org/grpc/test/grpc_testing"
+	testpb "google.golang.org/grpc/test/grpc_testing"
+)
+
+// CheckRPCsToBackend makes a bunch of RPCs on the given ClientConn and verifies
+// if the RPCs are routed to a peer matching wantAddr.
+//
+// Returns a non-nil error if context deadline expires before all RPCs begin to
+// be routed to the peer matching wantAddr.
```

Comment:
Also returns errors on line 65 (error in RPC) and line 68 (error in expectation for which addr the peer was). Perhaps document?

Reply:
Done.